src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Utilities/TypeStyle/CSharpUseImplicitTypeHelper.cs

```diff
@@ -225,7 +211,48 @@ protected override bool IsStylePreferred(in State state)
             var newDeclarationTypeNode = newTree.GetRoot(cancellationToken).GetAnnotatedNodes(annotation).Single();
             var newDeclarationType = newSemanticModel.GetTypeInfo(newDeclarationTypeNode, cancellationToken).Type;
 
-            return SymbolEquivalenceComparer.Instance.Equals(declarationType, newDeclarationType);
+            return SymbolEquivalenceComparer.TupleNamesMustMatchInstance.Equals(
+                declarationType, newDeclarationType);
+        }
+
+        private bool IsMatchingArgToSimpleMethod(DeclarationExpressionSyntax declarationExpression, SemanticModel semanticModel, CancellationToken cancellationToken)
```

Comment:
I'll be honest I have no idea what "IsMatchingArgToSimpleMethod" means.

Reply:
Fair.  The comments inside try to explain what's going on... Do you hve a better name?

Reply:
done.

Reply:
Like the new name, thanks!